python/couler/couler/pyfunc.py

```diff
@@ -71,9 +71,12 @@ def body(func_obj):
     return textwrap.dedent(code)
 
 
-def workflow_filename():
-    """Return the Python file that defines the workflow.
+def workflow_name():
+    """Return the workflow name that defines the workflow.
     """
+    wf_name = os.getenv("workflow_name")
+    if wf_name != "":
+        return wf_name
     stacks = inspect.stack()
     frame = inspect.stack()[len(stacks) - 1]
     full_path = frame[0].f_code.co_filename
```

Comment:
This function still returns filename if env `workflow_name` does not exist, so the function name `workflow_name` may not telling this detail?